### PR TITLE
issue #8179 from video.js repo - Updated getting-started.mdx

### PR DESCRIPTION
**Pull request by ravixalgorithm**
issue [#8176](https://github.com/videojs/video.js/issues/8179) resolved from [video.js](https://github.com/videojs/video.js)

Updated getting-started.mdx

### CODE REVIEW

**Comment by ravixalgorithm** [comment]
@mister-ben I did the changes

**Comment by ravixalgorithm** [comment]
@mister-ben can you pls review this PR and merge it??

File: src/mdx-pages/getting-started.mdx [hunk 1]
```diff
@@ -96,7 +96,29 @@ Video.js/
 └── video.min.js
 ```
 
-This package includes everything you'll need to use Video.js on a production site. By default, we bundle Video.js with Mozilla's excellent [VTT.js](https://github.com/mozilla/vtt.js). If you don't need VTT.js functionality for whatever reason, you can use one of the Video.js copies that don't include VTT.js. These have <code>novtt</code> in the name and can be found in the <code>alt/</code> directory. <code>font/</code> includes all the generated icon font files from the [Videojs Font project](https://github.com/videojs/font). <code>lang/</code> contains all the generated translation files.
+This package includes everything you'll need to use Video.js on a production site.
+
+#### Explanation of Core Versions
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:101]:
```suggestion
#### Explanation of the Versions
```

File: src/mdx-pages/getting-started.mdx [hunk 2]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:102]:
```suggestion

- **video.js**: The default build of Video.js, which includes the [HTTP Streaming (VHS)](https://github.com/videojs/http-streaming) playback engine for HLS and DASH playback with [Media Source Extensions](https://developer.mozilla.org/en-US/docs/Web/API/Media_Source_Extensions_API) and [our fork of Mozilla's VTT.js](https://github.com/videojs/vtt.js/).
```

File: src/mdx-pages/getting-started.mdx [hunk 3]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.core.js**: The core version of Video.js without any plugins or additional features.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:103]:
```suggestion
- **alt/video.core.js**: The core version of Video.js without VHS.
```

I would include the path to reinforce where to find these.

File: src/mdx-pages/getting-started.mdx [hunk 4]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.core.min.js**: The minified version of the core Video.js file, optimized for production use.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:104]:
```suggestion
```

It's probably easier to scan if min is just mentioned separately at the end.

File: src/mdx-pages/getting-started.mdx [hunk 5]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.core.novtt.js**: The core version of Video.js without VTT.js, suitable if you don't need VTT.js functionality.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:105]:
```suggestion
- **alt/video.core.novtt.js**: The core version of Video.js without VTT.js, suitable if you don't need VTT.js functionality.
```

File: src/mdx-pages/getting-started.mdx [hunk 6]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.core.novtt.min.js**: The minified version of the core Video.js file without VTT.js.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:106]:
```suggestion
```

File: src/mdx-pages/getting-started.mdx [hunk 7]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.novtt.js**: A version of Video.js with additional features but without VTT.js.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:107]:
```suggestion
- **alt/video.novtt.js**: A version of Video.js with VHS but without VTT.js.
```

File: src/mdx-pages/getting-started.mdx [hunk 8]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.novtt.min.js**: The minified version of Video.js with additional features but without VTT.js.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:108]:
```suggestion

Each file is also available as a `.min.js` version for production which is minified with uglify-js.
```

File: src/mdx-pages/getting-started.mdx [hunk 9]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+- **video.core.js**: ~943KB
+- **video.core.min.js**: ~252KB
+- **video.core.novtt.js**: ~886KB
+- **video.core.novtt.min.js**: ~233KB
+- **video.novtt.js**: ~2,225KB
+- **video.novtt.min.js**: ~629KB
+- **video.js**: ~2,282KB
+- **video.min.js**: ~648KB
+
+By default, we bundle Video.js with Mozilla's excellent [VTT.js](https://github.com/mozilla/vtt.js). If you don't need VTT.js functionality for whatever reason, you can use one of the Video.js copies that don't include VTT.js. These have <code>novtt</code> in the name and can be found in the <code>alt/</code> directory. <code>font/</code> includes all the generated icon font files from the [Videojs Font project](https://github.com/videojs/font). <code>lang/</code> contains all the generated translation files.
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:121]:
```suggestion
<code>font/</code> includes all the generated icon font files from the [Videojs Font project](https://github.com/videojs/font). <code>lang/</code> contains all the generated translation files.
```

The part about VTT is redundant with the bullet points above.

File: src/mdx-pages/getting-started.mdx [hunk 10]
```diff
@@ -96,7 +96,29 @@ Video.js/
...
+#### Approximate File Sizes
+
+- **video.core.js**: ~943KB
+- **video.core.min.js**: ~252KB
+- **video.core.novtt.js**: ~886KB
+- **video.core.novtt.min.js**: ~233KB
+- **video.novtt.js**: ~2,225KB
+- **video.novtt.min.js**: ~629KB
+- **video.js**: ~2,282KB
+- **video.min.js**: ~648KB
```
Comment (mister-ben) [inline src/mdx-pages/getting-started.mdx:119]: I think we probably don't need this. File sizes change slightly all the time. The sizes for each published version can be viewed at https://www.npmjs.com/package/video.js?activeTab=code

**Review by mister-ben — CHANGES_REQUESTED** [review]
Thanks! Some suggestions inline.

**Comment by ravixalgorithm** [comment]
I updated @mister-ben as you instructed

**Review by ravixalgorithm — COMMENTED** [review]
I did all the changes as instructed by @mister-ben

File: src/mdx-pages/getting-started.mdx (hunk 10 above)
Comment (ravixalgorithm) [reply src/mdx-pages/getting-started.mdx]: did the changes

**Comment by mister-ben** [comment]
We need to make some CSS changes as the bullet list doesn't look quite right. It's not the fault of these changes, it will just take a little longer to get in.

**Pull request closed without merging**